src/example-types/metricGroups/dateIntervalGroupStats.js

```diff
@@ -1,22 +1,64 @@
-let moment = require('moment-timezone')
+let moment = require('moment')
 let { groupStats } = require('./groupStatUtils')
 
 let drilldown = ({ field, interval, drilldown }) => {
+  if (interval === 'federalFiscalYear') interval = 'year'
+  if (interval === 'federalFiscalQuarter') interval = 'quarter'
   let gte = drilldown
   let lte = moment
     .parseZone(drilldown)
     .endOf(interval)
     .format()
   return { range: { [field]: { gte, lte } } }
 }
-let buildGroupQuery = ({ field, interval = 'year' }, children, groupsKey) => ({
-  aggs: {
-    [groupsKey]: {
-      date_histogram: { field, interval, min_doc_count: 0 },
-      ...children,
+
+let isOffsetDates = node => {
+  if (
+    node.interval === 'federalFiscalYear' ||
+    node.interval === 'federalFiscalQuarter'
+  )
+    return true
+  return false
+}
```

Comment:
Functions that return true/false instead of the condition is a personal pet peeve 😅 

This can be reduced to

```js
let isOffsetDates = ({ interval }) =>
  interval === 'federalFiscalYear' || interval === 'federalFiscalQuarter'
```

or even just:

```js
let isOffsetDates = node => _.includes('federalFiscal', node.interval)
```

Side note -  I don't love the `federalFiscal` prefix as part of `contexture-elasticsearch` as it's a little domain-specific. I think dropping `federal` and just having a `fiscal` prefix is likely fine (especially since we plan to parameterize the month offset).